x-pack/plugins/integrations_manager/index.ts

```diff
@@ -14,7 +14,7 @@ import {
   Server,
   ServerPluginInitializerContext,
 } from './common/types';
```

Comment:
`CoreSetup` from `src/core/server` is different than the one from `src/core/public`, you're using the correct types in all your code, but exporting this type from `./common/types` might cause some confusion down the line.

Reply:
I totally agree. Thanks. I moved a bunch of stuff out of `common/types` in https://github.com/elastic/kibana/pull/39438 Take a look and LMK what you think.